src/app/personside/visittkort/body/telefon/Mobiltelefon.tsx

```diff
@@ -0,0 +1,47 @@
+import * as React from 'react';
+import Undertekst from 'nav-frontend-typografi/lib/undertekst';
+
+import VisittkortElement from '../VisittkortElement';
+import { Kontaktinformasjon } from '../../../../../models/kontaktinformasjon';
+import Innholdslaster from '../../../../../components/Innholdslaster';
+import { Reducer } from '../../../../../redux/reducer';
+import EtikettLiten from 'nav-frontend-typografi/lib/etikett-liten';
+
+const phonePath = require('../../../../../resources/svg/phone.svg');
+
+interface MobiltelefonProps {
+    kontaktinformasjon: Kontaktinformasjon;
+}
+
+function Mobiltelefon({kontaktinformasjon }: MobiltelefonProps) {
+    if (kontaktinformasjon.reservert) {
+        return (
+            <Undertekst>Reservert</Undertekst>
+        );
+    } else if (kontaktinformasjon.mobiltelefon) {
+        return (
+            <>
+                <Undertekst>{kontaktinformasjon.mobiltelefon.value}</Undertekst>
```

Comment:
Bør vi kanskje legge inn nummergruppering her et sted? Altså «999 99 999» i stedet for «99999999».

Reply:
Enig i det egentlig. Det er ikke det i skissene, men det ser jo mye bedre ut når det er formatert.

Reply:
Lagde en issue på det: https://github.com/navikt/modiapersonoversikt/issues/58